### PR TITLE
Will/projects list

### PR DESCRIPTION
**Pull request by wdeucher**
Used Liz's nav bar as template. Link works from the homepage to the Projects List page. Not sure if this PR should be merged with liz/nav_bar or master.

Still need to upload the correct font family. I found it on google fonts, now I just need to integrate it. That should be able to be done once for all of the pages.

### CODE REVIEW

File: .remote-sync.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (MichaelScottPaperCompany) [inline .remote-sync.json:1]: @wdeucher could you add this file to the `.gitignore`?

File: README.md [hunk 2]
```diff
@@ -1,14 +1,4 @@
-# Tufts CS Code For Good Website
```
Comment (MichaelScottPaperCompany) [inline README.md:1]: Why did you remove the rest of the readme?

File: pages/funding.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        <div className={styles.column}>
+
+        <div className={styles.nav_bar}>
+           <a href ="/"><img src="/images/TempLogo.jpg" alt="Tufts Code For
+           Good Logo" width= "181px" height="61px" border="solid 1px #979797"/>
+           </a>
+           <a href="/about" className={styles.nav_bar}>ABOUT</a>
+           <a href="/projects" className={styles.nav_bar}>PROJECTS</a>
+           <a href="/funding" className={styles.nav_bar}>FUNDING</a>
+         </div>
```
Comment (MichaelScottPaperCompany) [inline pages/funding.js:16]: I think this stuff can go in the `_app.js` file. I'm working on an example now

File: pages/projects.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+          <div className={s.item}>
+            <img src="/images/projects_logos/example2.jpg" alt="2" className={s.logo} />
+          </div>
+          <div className={s.item}>
+            <img src="/images/projects_logos/example1.jpg" alt="1" className={s.logo} />
+          </div>
+          <div className={s.item}>
+            <img src="/images/projects_logos/example3.jpg" alt="3" className={s.logo} />
+          </div>
+        </div>
```
Comment (MichaelScottPaperCompany) [inline pages/projects.js:70]: it would be sweet to dynamically render the list from a `.json` file. You can import the file directly using the `import projects from '../projects.json';` and then pull the correct properties like name and link from that file. We can use the same JSON to populate the project-specific page.

File: pages/projects.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+      
+        <div id={s.text_container}>
+          <h1 className={s.txt} id={s.title}>PROJECTS</h1>
+          <h3 className={s.txt} id={s.sub}>TUFTS CODE FOR GOOD</h3>
+        </div>
+        <div className={s.column}>
```
Comment (MichaelScottPaperCompany) [inline pages/projects.js:27]: For some reason the whole page doesn't scroll down for me. I'm not exactly sure what the issue is but I think it's a CSS problem with the main container. I'm looking into it.

Comment (wdeucher) [reply pages/projects.js:27]: I changed the background's position value from 'relative' to 'absolute' and the scrolling is fixed now
